packages/jobs/lib/activities.ts

```diff
@@ -98,6 +98,7 @@ export async function runAction(args: ActionArgs): Promise<ServiceResponse> {
         orchestratorClient,
         logContextGetter,
         writeToDb: true,
+        logCtx: await logContextGetter.get({ id: String(activityLogId) }),
```

Comment:
yes because we still create other operation for webhook and slack if I remember well, definitely not related but until better it's there :p

Reply:
indeed the logContextGetter is passed to the `slackNotificationService` function. Since the `slackNotificationService` is also passed as a param can we give the `logContextGetter` to the `slackNotificationService` constructor and avoid having to pass it down to each of its methods. That I think would remove the need to pass it to the run service.

Reply:
SlackService is not passed as a param to runService. But I realised orchestrator was also passed just for slack, so I tried a bigger change, let me know: 1d41fd4e136f18b484a86c36f6ffb70c51a8a491

Reply:
I love it. I should have done that instead of injecting the orchestratorClient